src/app/components/Header/HeaderSearchBar/HeaderSearchBar.tsx

```diff
@@ -50,16 +50,16 @@ export const HeaderSearchBar: FC<HeaderSearchBarProperties> = ({
 		}
 	}, [resetFields, handleQueryReset]);
 	return (
-		<div data-testid="HeaderSearchBar" className="-my-2" {...properties}>
+		<div data-testid="HeaderSearchBar" className="relative -my-2" {...properties}>
```

Comment:
@patricio0312rev I had to restore this as without `relative` almost all search inputs are misplaced